package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@nypl/dgx-svg-icons": "^0.3.9",
     "bootstrap": "^3.3.6",
-    "library-simplified-reusable-components": "1.3.7",
+    "library-simplified-reusable-components": "^1.3.7",
```

Comment:
Why the caret?

Reply:
I don't like using carets because they can create dependencies issues if any new packages.

Reply:
It was suddenly breaking otherwise.  No idea why.

Reply:
I know, but...this seems like a reasonable exception, given that it's our package.  Does it work for you without the caret?

Reply:
It's not breaking and it should be fine with our package, but in general for other packages we don't use. That reminds me, it seems an updated package.json should be committed (a new version comes up when I run `npm install`).

Reply:
And...it's working without the caret again.  It's a mystery.  That's...nice.

Reply:
Maybe an issue with the updated package-lock file.